### ISSUE TITLE
ianitor OSError: [Errno 22] Invalid argument

### ISSUE THREAD

**Issue by thijsterlouw** [ClearcodeHQ/ianitor#2]
```
 ianitor --consul-agent=127.0.0.1:8500 test -- bash
```

output:
```
Traceback (most recent call last):
  File "/home/thijsterlouw/.pyenv/versions/myenv/bin/ianitor", line 9, in <module>
    load_entry_point('ianitor==0.0.1', 'console_scripts', 'ianitor')()
  File "/home/thijsterlouw/.pyenv/versions/myenv/lib/python3.4/site-packages/ianitor/script.py", line 88, in main
    signal.signal(signum, signal_handler)
OSError: [Errno 22] Invalid argument
```

in the consul server:
```
    2015/03/12 11:18:20 [INFO] agent: Synced service 'test'
    2015/03/12 11:18:20 [INFO] agent: Deregistered service 'test'
    2015/03/12 11:18:20 [INFO] agent: Deregistered check 'service:test'
```

whatever service I attempt to run, I always receive the error "OSError: [Errno 22] Invalid argument"

On Linux / Kubuntu 14.04, Python 3.4.3 in virtualenv

**Comment by swistakm** [comment]
This is caused by `SIGSTOP` signal raising OSError exception when trying to bind it on python3.4. Unfortunately `ianitor.script` module was not tested at all :(.

**Comment by swistakm** [comment]
New version with fix for this issue released on PyPI: 0.0.2
